ts/parts/Axis.ts

```diff
@@ -1761,6 +1761,30 @@ extend(Axis.prototype, /** @lends Highcharts.Axis.prototype */{
             }
         },
 
+        /**
+         * The left position as the horizontal axis. If it's a number, it is
+         * interpreted as pixel position relative to the chart.
+         *
+         * Since Highstock 2: If it's a percentage string, it is interpreted
```

Comment:
`left` and `width` are supporting percentage since v5.0.13 ([changelog](https://www.highcharts.com/blog/changelog/#highstock-v5.0.13)).